src/Nethermind/Nethermind.Optimism/OptimismBaseFeeCalculator.cs

```diff
@@ -31,8 +31,8 @@ public UInt256 Calculate(BlockHeader parent, IEip1559Spec specFor1559)
             spec = eip1559Params.IsZero()
                 ? new OverridableEip1559Spec(specFor1559)
                 {
-                    ElasticityMultiplier = Eip1559Constants.DefaultElasticityMultiplier,
-                    BaseFeeMaxChangeDenominator = Eip1559Constants.DefaultBaseFeeMaxChangeDenominator
+                    ElasticityMultiplier = spec.ElasticityMultiplier,
+                    BaseFeeMaxChangeDenominator = spec.BaseFeeMaxChangeDenominator
```

Comment:
The defaults that should be used should come from ChainSpec rather than from constants. During initialization we deal with other forks like Canyon, and if no values are provided in ChainSpec then we'll load the default constants.